code/_globalvars/global_lists.dm

```diff
@@ -14,6 +14,20 @@ GLOBAL_LIST_EMPTY(failed_fultons) //A list of fultoned items which weren't colle
 GLOBAL_LIST_INIT_TYPED(custom_huds_list, /datum/custom_hud, setup_all_huds())
 GLOBAL_LIST_INIT_TYPED(custom_human_huds, /datum/custom_hud, setup_human_huds())
 
+//Basically counts how many times the Almayer was hit by each weapon. Loggin' wack.
+GLOBAL_LIST_INIT(ship_hits_tally, list(
+"times_hit_missile" = 0,
+"times_hit_railgun" = 0,
+"times_hit_odc" = 0,
+"times_hit_aaboiler" = 0,
```

Comment:
This would be a place to re-use your weapon definition (but will require you to define them in a code_DEFINES file and not undefed. E.g.
```
GLOBAL_LIST_INIT(ship_hits_tally, list(
	SVS_WEAPON_MISSILE = 0,
	SVS_WEAPON_RAILGUN = 0,
	SVS_WEAPON_ODC = 0,
	SVS_WEAPON_AABOILER = 0,
))
```